lightning/src/ln/functional_test_utils.rs

```diff
@@ -969,19 +983,18 @@ macro_rules! get_route_and_payment_hash {
 }
 
 macro_rules! expect_pending_htlcs_forwardable_ignore {
-	($node: expr) => {{
-		let events = $node.node.get_and_clear_pending_events();
-		assert_eq!(events.len(), 1);
-		match events[0] {
+	($node: expr, $events: expr) => {{
```

Comment:
This generates a lot of diff - maybe we can add an `expect_pending_htlcs_forwardable_from_events` or have a dual-form macro?

Reply:
Done.